### PR TITLE
Adding unit tests for device.go

### PR DESCRIPTION
**Pull request by lidiyag**
**What type of PR is this?**
 /kind test

**What this PR does / why we need it**:
This PR adds unit tests for edge/pkg/devicetwin/dtmanager/device.go

### CODE REVIEW

**Comment by lidiyag** [comment]
cc @rohitsardesai83 , @sids-b , @m1093782566

**Comment by m1093782566** [comment]
When can we add UTs for cloud part?

File: edge/pkg/devicetwin/dtmanager/device_test.go [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+				DTContexts:   dtContextStateConnected,
+			},
+		},
+	}
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			dw := DeviceWorker{
+				Worker: test.Worker,
+			}
+			go dw.Start()
```
Comment (sids-b) [inline edge/pkg/devicetwin/dtmanager/device_test.go]: Please assert in the test case.

Comment (lidiyag) [reply edge/pkg/devicetwin/dtmanager/device_test.go]: Updated..Please see..thanks

**Pull request merged**